sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/FunctionCallingSample.java

```diff
@@ -26,11 +28,11 @@
 import java.util.Map;
 
 /**
- * Sample demonstrates how to get chat completions using function call.
+ * Sample demonstrates how to get chat completions using function calling.
  */
-public class FunctionCallSample {
+public class FunctionCallingSample {
```

Comment:
nit: I think that `FunctionCall` was better :D as it points the user directly to the class and it's not necessarily grammatically incorrect. 

nit 2: I would go as far as maybe renaming to `ChatCompletionsFunctionCallSample` to really drive the point that this is a `ChatCompletions` feature.